client/src/common/model/actions.js

```diff
@@ -20,9 +20,12 @@ export const joinRoom = (route, id, userId) => {
   socket.emit(AppEvents.LEAVE_ROOM, metaDataChannel(id, route));
 };
 
-export const leaveRoom = (route, id, userId) => {
+export const leaveRoom = (route, id, userId) => isDisabled => {
```

Comment:
it is hard to tell what is happening when calling this method - please initialize with object with named parameters.